dev-utils/karma.js

```diff
@@ -142,11 +142,6 @@ function prepareConfig(defaultConfig) {
         ]
       }
     }
-  } else {
```

Comment:
I will refactor this later to enable coverage run on local as well. Let's do it in separate PR not really important though.

Reply:
No issues though, while fixing bugs, Victor moved it to jenkins since the coverage is only run there. But I thought we can do that later as well.

Reply:
I will refactor it now.